### PR TITLE
finalizing 'buttons' style guide page

### PR DESCRIPTION
**Pull request by indirectlylit**
I did a pass of editing on the buttons page. Once this is done, I think we'll be able to have our first officially **complete** style guide page!

There were a number of small tweaks, but the substantive changes are outlined below:

- also includes code examples for hyperlinks
- added inline examples, separate from the interactive code example
- moved info related to 'scope' and 'exceptions' to style guide home page because the guidance applies to all components
- added note: "Buttons can also be primary or secondary. There should only be one primary button visible at a time, and it should be a common or default action."
- removed dropdown button info - will be migrated to a new component
- removed sections that are embodied in the implementation
  - color usage
  - responsiveness
- added placement guidance: "left-aligned in page forms"
- moved "no icon" guidance to "Language and contents" section
- removed duplicated sections
- added "known exceptions" section

## After

localhost-8000-style_guide 4

## Before

localhost-8000-style_guide 1

### CODE REVIEW

File: kolibri/plugins/style_guide/assets/src/views/content/buttons/index.vue [hunk 1]
```diff
@@ -1,117 +1,68 @@
...
-      There are 3 main types of buttons, each to map to a different level of
-      visual hierarchy
+      There are 3 main types of buttons. Each maps to a different level of
+      visual hierarchy:
     </p>
     <ul>
-      <li>Raised buttons for more prominent actions</li>
-      <li>Flat buttons for less prominent or actions that repeat in a list</li>
-      <li>Hyperlink text for very deemphasized actions or in paragraphs.</li>
+      <li>Raised buttons: more prominent actions</li>
```
Comment (christianmemije) [inline kolibri/plugins/style_guide/assets/src/views/content/buttons/index.vue]: Bolding the 3 types would be helpful

File: kolibri/plugins/style_guide/assets/src/views/content/buttons/index.vue [hunk 2]
```diff
@@ -1,117 +1,68 @@
...
-    <p>
-      Buttons should aim to have a single word in them, with two at most
-    </p>
     <p>
-      The buttons language should avoid any ambiguity, be specific to the
-      action you are completing. For example, use 'Save' instead of 'OK'.
-    </p>
-    <p>
-      Never use commas or other punctuation in buttons
+      Buttons can also be primary or secondary. There should only be one primary button visible at a time, and it should be a common or default action.
```
Comment (christianmemije) [inline kolibri/plugins/style_guide/assets/src/views/content/buttons/index.vue]: Bold `primary` and `secondary`

File: kolibri/plugins/style_guide/assets/src/views/shell/page-template.vue [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <template>
 
-  <div>
-    <h1>{{ title }}</h1>
-    <p v-if="!completed">(In progress)</p>
+  <div class="template-wrapper">
+    <p v-if="!completed" class="in-progress">IN PROGRESS</p>
```
Comment (christianmemije) [inline kolibri/plugins/style_guide/assets/src/views/shell/page-template.vue:4]: Nice

**Comment by khangmach** [comment]
looks good, but what does elide refer to in the responsive section? I can't think of any reason why buttons should merge?

**Comment by indirectlylit** [comment]
thanks!

To clarify for future reference: the original content said that "buttons should elide if they need to", meaning that if a label is too long then it should be truncated with a '...'

I removed that text because it's not embodied in the implementation.

**Pull request merged**